cmd/soci/commands/index/list.go

```diff
@@ -39,17 +39,23 @@ func originalDigestFilter(digest string) func(*soci.ArtifactEntry) bool {
 }
 
 var listCommand = cli.Command{
-	Name:  "list",
-	Usage: "list indices",
+	Name:    "list",
+	Usage:   "list indices",
+	Aliases: []string{"ls"},
 	Flags: []cli.Flag{
 		cli.StringFlag{
 			Name:  "ref",
 			Usage: "filter indices to those that are associated with a specific image ref",
 		},
+		cli.BoolFlag{
```

Comment:
Doesn't have to be part of this PR, but this should allow us to get rid of the `soci image list-indices`. It's used by integ tests since it just returns the digest and I never got around to replacing it with `soci index list` because I wanted an easier-to-parse output format.